fmriprep/cli/run.py

```diff
@@ -147,9 +147,7 @@ def create_workflow(opts):
     make_folder(settings['output_dir'])
     make_folder(settings['work_dir'])
 
-    if opts.reports_only:
-        run_reports(settings['output_dir'])
-        sys.exit()
+
```

Comment:
This is now three lines between `make_folder(settings['work_dir'])` and `# nipype plugin configuration`. Can you reduce it to one?